hardware_interface/include/hardware_interface/hardware_info.hpp

```diff
@@ -24,88 +24,88 @@ namespace hardware_interface
 {
 
 /**
- * \brief This structure stores information about components defined for a specific hardware
+ * /// This structure stores information about components defined for a specific hardware
```

Comment:
```suggestion
/// This structure stores information about components defined for a specific hardware
```
this looks odd. One line comments should only be with `///`

Reply:
same below.